api/Animation.json

```
@@ -177,6 +177,42 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "endpoint_inclusive_timing": {
+          "__compat": {
+            "description": "animation `fill` mode not needed",
```

As proposed by spec author

```suggestion
        "endpoint_inclusive_commitStyles": {
          "__compat": {
            "description": "`commitStyles()` automatically fills values",
```